docs/rules/README.md

```diff
@@ -27,41 +27,41 @@ The following rules point out areas where you might have made mistakes.
-* [no-regex-spaces](no-regex-spaces.md): disallow multiple spaces in a regular expression literal (recommended)
+* [no-negated-in-lhs](no-negated-in-lhs.md): disallow the negation of the left operand in an `in` operator (recommended)
+* [no-obj-calls](no-obj-calls.md): disallow the use of object properties of the global object as functions (recommended)
+* [no-regex-spaces](no-regex-spaces.md): disallow multiple spaces in regular expression literals (recommended)
 * [no-sparse-arrays](no-sparse-arrays.md): disallow sparse arrays (recommended)
-* [no-unexpected-multiline](no-unexpected-multiline.md): disallow code that looks like two expressions but is actually one (recommended)
-* [no-unreachable](no-unreachable.md): disallow unreachable statements after a return, throw, continue, or break statement (recommended)
-* [use-isnan](use-isnan.md): disallow comparisons with the value `NaN` (recommended)
-* [valid-jsdoc](valid-jsdoc.md): ensure JSDoc comments are valid
-* [valid-typeof](valid-typeof.md): ensure results of typeof are compared against a valid string (recommended)
+* [no-unexpected-multiline](no-unexpected-multiline.md): disallow unexpected multiline expressions (recommended)
+* [no-unreachable](no-unreachable.md): disallow unreachable code (recommended)
+* [use-isnan](use-isnan.md): enforce the use of `isNaN()` when comparing against `NaN` (recommended)
+* [valid-jsdoc](valid-jsdoc.md): enforce valid JSDoc comments
+* [valid-typeof](valid-typeof.md): ensure results of `typeof` are compared against valid strings (recommended)
 
 ## Best Practices
 
 These are rules designed to prevent you from making mistakes. They either prescribe a better way of doing something or help you avoid footguns.
 
-* [accessor-pairs](accessor-pairs.md): enforce getter/setter pairs in objects
-* [array-callback-return](array-callback-return.md): enforce return statements in callbacks of array's methods
+* [accessor-pairs](accessor-pairs.md): enforce getter and setter pairs in objects
+* [array-callback-return](array-callback-return.md): enforce `return` statements in callbacks of array methods
 * [block-scoped-var](block-scoped-var.md): treat `var` statements as if they were block scoped
 * [complexity](complexity.md): specify the maximum cyclomatic complexity allowed in a program
 * [consistent-return](consistent-return.md): require `return` statements to either always or never specify values
 * [curly](curly.md): specify curly brace conventions for all control statements
-* [default-case](default-case.md): require `default` case in `switch` statements
-* [dot-location](dot-location.md): enforce consistent newlines before or after dots
-* [dot-notation](dot-notation.md): encourage use of dot notation whenever possible
-* [eqeqeq](eqeqeq.md): require use of `===` and `!==`
-* [guard-for-in](guard-for-in.md): ensure `for-in` loops have an `if` statement
+* [default-case](default-case.md): enforce existence of `default` case in `switch` statements
```

Comment:
I actually prefer the word `require` - I know we use `enforce` in a lot of places, but I think `require` is clearer and more succinct than the above